example/App.js

```diff
@@ -19,8 +19,8 @@ import ImageStack from './src/ImageStack';
 import TransparentStack from './src/TransparentStack';
 import ModalStack from './src/ModalStack';
 import ModalPresentation from './src/ModalPresentation';
-import LifecycleInteraction from './src/LifecycleInteraction';
-import GestureInteraction from './src/GestureInteraction';
+//import LifecycleInteraction from './src/LifecycleInteraction';
```

Comment:
sth is wrong with expo. I don't really have time for focusing on it. We'll revert this example after new release of expo

Reply:
the problem is that we're importing from the 'expo' package here, need to import from 'expo-barcode-scanner' and 'react-native-maps' and also give a better error when importing from the 'expo' package in a vanilla react-native project :)